lambdas/data-migration2/tests/test-executions.js

```diff
@@ -149,7 +149,7 @@ test.serial('migrateExecutionRecord correctly migrates execution record', async
   t.context.collectionCumulusId = pgCollection.cumulus_id;
 
   const asyncOperationPgModel = new AsyncOperationPgModel();
-  const [asyncOperationCumulusId] = await asyncOperationPgModel.create(
+  const [pgAsyncOperation] = await asyncOperationPgModel.create(
```

Comment:
I take it this wasn't passing. Do we not run this test in. CI?

Reply:
The default base.create() method returns `cumulus_id`.  With knex 0.95, it returns the number, knex 2 returns [{cumulus_id: thevalue}].
https://github.com/nasa/cumulus/blob/master/packages/db/src/models/base.ts#L229

Reply:
OH! Yikes.